geomesa-hbase/geomesa-hbase-spark-runtime/src/test/scala/org/locationtech/geomesa/hbase/spark/HBaseSparkProviderTest.scala

```diff
@@ -0,0 +1,113 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.hbase.spark
+
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.spark.sql.{DataFrame, SQLContext, SQLTypes, SparkSession}
+import org.geotools.data.{Query, Transaction}
+import org.geotools.factory.CommonFactoryFinder
+import org.junit.runner.RunWith
+import org.locationtech.geomesa.hbase.data.HBaseDataStoreFactory
+import org.locationtech.geomesa.hbase.data.HBaseDataStoreFactory.Params._
+import org.locationtech.geomesa.spark.SparkSQLTestUtils
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+import scala.collection.JavaConversions._
+
+@RunWith(classOf[JUnitRunner])
+class HBaseSparkProviderTest extends Specification with LazyLogging {
+
+  // START HBASE INSTANCE MANUALLY
```

Comment:
should use an embedded hbase here, or rename this to `HBaseSparkProviderIntegrationTest` and skip it during the normal build